### ISSUE TITLE
Padding Inconsistencies with layout-padding

### ISSUE THREAD

**Issue by langdonx** [angular/material#4356]
In making some responsive layouts, I ran into an odd padding issue.

**tldr:**
 - looks fine: http://jsfiddle.net/langdonx/pun3h4a9/
 - change `flex-gt-lg` to `flex-gt-md`: http://jsfiddle.net/langdonx/pun3h4a9/1/
  - now headline 1, 2, and 3 have more padding than they should

**Analysis:**

There are 3 padding scenarios when it comes to using `layout-padding` (currently 16px, 8px, and 4px based on a current `$layout-gutter-width` of 8).  At first glance, it seems like `flex-gt-lg` should probably be included along side `flex-gt` to get 16px padding, but maybe the `gt-lg` layout doesn't warrant any padding?

Source reference: https://github.com/angular/material/blob/2fe726c536388c1033b8180c1cb6eb429d0915fe/src/core/services/layout/layout.scss#L32

Ultimately I feel like these should be media queries, and not based on element attributes, however I just got started, so maybe there's something I'm not aware of.

**Comment by tklepzig** [comment]
+1

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our deprecation effort.
    For details, see our post [Spring Cleaning 2016](https://groups.google.com/forum/#!topic/ngMaterial/3Ip9nd68bfk).

**Comment by ThomasBurleson** [comment]
This issue is closed as part of our deprecation effort.
    For details, see our post [Spring Cleaning 2016](https://groups.google.com/forum/#!topic/ngMaterial/3Ip9nd68bfk).
